tests/Unit/Evolution/DiscontinuousGalerkin/Actions/Test_ApplyBoundaryCorrections.cpp

```diff
@@ -81,12 +81,10 @@ struct BoundaryTerms final : public BoundaryCorrection<Dim> {
     using type = Scalar<DataVector>;
   };
 
-  /// \cond
-  explicit BoundaryTerms(CkMigrateMessage* /*unused*/) noexcept {}
+    explicit BoundaryTerms(CkMigrateMessage* /*unused*/) noexcept {}
```

Comment:
why is the indentation changing in some of these files?

Reply:
please fix the indentation changes in a separate fixup commit to make it easier to check that you did them

Reply:
Good catch! In the regex-replace I didn't include the whitespace preceding the `/// \cond`, so it got added to the following line. I repeated the regex-replace and pushed the diff as a fixup.